app/code/Magento/CatalogImportExport/Model/Import/Product/Type/AbstractType.php

```diff
@@ -476,6 +476,8 @@ public function isRowValid(array $rowData, $rowNum, $isNewProduct = true)
                 // check value for non-empty in the case of required attribute?
                 if (isset($rowData[$attrCode]) && strlen($rowData[$attrCode])) {
                     $error |= !$this->_entityModel->isAttributeValid($attrCode, $attrParams, $rowData, $rowNum);
+                } elseif(isset($rowData['product_type']) && !in_array($rowData['product_type'], $attrParams['apply_to'])){
```

Comment:
Hi @woutersamaey. Thanks for collaboration. Could you explain the reason to add empty condition? Maybe, you should the next condition more strict instead of create new empty one?

Reply:
Just wanted to make sure the needed data is there.
I’m not sure the product_type is required.
We for example use this import logic in more situations, not just for CSV importing.